### ISSUE TITLE
[OPS] Difficulty to edit fields in other languages

### ISSUE THREAD

**Issue by alexxxmendonca** [pkp/pkp-lib#6039]
**Describe the bug**
When I try to edit a field in a specific language other than English, the interface is always split in half, which makes it really difficult to edit the content, not to mention that the source HTML button is not visible:

![image](https://user-images.githubusercontent.com/4059679/85568049-be2f0400-b607-11ea-9344-c17cfd5047ad.png)

Even if I change the interface language at the language toggle button, the English is fixed and always appearing.

**To Reproduce**
1. Go to Settings -> Website -> Appearance -> Advanced -> Additional Content
2. Notice the English content.
3. Change interface to another other than English (top right language toggle button)
4. Repeat setp 1.
5. Notice how the interface is split in two showing both English and the active language, making it really hard to edit content.

**What application are you using?**
OPS v3.2.1

**Additional information**
![Website Settings](https://user-images.githubusercontent.com/4059679/85569396-e408d880-b608-11ea-87d9-8382e68fa120.gif)

**Comment by NateWr** [comment]
This pattern was implemented to prevent the problems we had before, where translation fields would cover up other fields, and to ensure that the translator could always see the source language they are translating from.

It does seem to work best for smaller fields rather than large textarea fields. Did you have an alternate layout in mind?

**Comment by alexxxmendonca** [comment]
True that the layout itself is not an issue. The main issue is that the source HTML button is hidden this way.

It's right after the image button and that's the last button we see. Everything after the image button is hidden, including the source HTML button. This makes it really hard to format the text.

**Comment by NateWr** [comment]
Ah ok, so the problem is not the amount of writing space that you have but the inability to access parts of the text editor toolbar. Got it. :+1:

**Comment by alexxxmendonca** [comment]
Right! It is a little inconvenient to have a small edit area, but it's still _doable_, whereas editing the HTML is not.

**Comment by ajnyga** [comment]
There is a scroll option for the toolbar it seems. https://www.tiny.cloud/docs/configure/editor-appearance/#scrolling
The other option is to use two rows of buttons and show the buttons in one row if they fit the screen.

**Comment by asmecher** [comment]
I believe this has already been resolved for 3.4, probably thanks to a TinyMCE update -- testing the `main` branch, when the toolbar's width exceeds the available area, the tool menu (or, if space permits, a piece of it) turns into a `...` menu:
![image](https://user-images.githubusercontent.com/200411/188997119-e5fd8757-1293-465d-aeaa-c2e77e3f80e5.png)

**Comment by NateWr** [comment]
Confirmed on my end. :tada:
